examples/failure-notification/versions/0.0.1/docs/failure-notification.md

```diff
@@ -0,0 +1,90 @@
+# failure-notifications
```

Comment:
singular to match template and file name

Reply:
fixed to show singular